Sprint_Tests/stepanov_maksym.md

```diff
@@ -0,0 +1,11 @@
+1. Previously, he was well-versed in markup and working on top-notch technologies at the frontend. After a few years of break and deep depression, plus listening to heavy music, he is incompetent in modern trends and drinks a lot. If he had run over depression, he wouldn’t have any problem found a job with a relevant salary. 
+
```

Comment:
cope with (run over usually means when somebody hits you with the car and doesn't stop)
any problems finding